test/versioned/koa/router-common.js

```diff
@@ -546,23 +546,20 @@ module.exports = (pkg) => {
         nestedRouter.get('/:second', function terminalMiddleware(ctx) {
           ctx.body = 'this is a test'
         })
-        nestedRouter.get('/second', function secondMiddleware(ctx) {
```

Comment:
I removed this because this middleare is never called but we had logic in `lib/instrumentation/koa/instrumentation.js` that was using this route name to name the transaction which was incorrect. I was also able to simplify that logic because we can just use what's being passed in to assign to the transaction namestate path